buildenv/jenkins/jobs/infrastructure/Cleanup-Nodes.groovy

```diff
@@ -109,7 +109,7 @@ timeout(time: TIMEOUT_TIME.toInteger(), unit: TIMEOUT_UNITS) {
                                  'testParentDir',
                                  'jni-*',
                                  'mauve',
-                                 'test*',
+                                 'test[^D]*',
```

Comment:
This will also leave `testDump` `testDiag`, etc. Can we instead expand the patterns and filter out `testDependency` instead?